### ISSUE TITLE
Using Meter is the exemplary integration with Dropwizard Metrics

### ISSUE THREAD

**Issue by leventov** [ben-manes/caffeine#235]
https://github.com/ben-manes/caffeine/blob/master/examples/stats-metrics/src/main/java/com/github/benmanes/caffeine/examples/stats/metrics/MetricsStatsCounter.java uses `Meter`, that is pretty heavyweight (note that it calls `System.nanoTime()` on every `mark()` that is [known to not scale very well](https://shipilev.net/blog/2014/nanotrusting-nanotime/#_backoff_measurements)). I think it would make more sense to provide a simpler windowed counter, with the window size configured in the `MetricsStatsCounter` rather than hard-coded, as in `Meter` (1/5/15 min).

**Comment by ben-manes** [comment]
Sure, if you'd like to send a PR to improve the example that would be great.

It used to be a best practice to record only incrementing values within the application, and let the aggregator perform any rollups. This is a lot simpler for all parties, and much more flexible. Is that no longer the case?

**Comment by ben-manes** [comment]
Oh, and of course you can use the [polling approach](https://github.com/finn-no/guava-metrics/blob/master/src/main/java/no/finn/metrics/GuavaCacheMetrics.java) if desired.

**Comment by leventov** [comment]
> It used to be a best practice to record only incrementing values within the application, and let the aggregator perform any rollups. This is a lot simpler for all parties, and much more flexible. Is that no longer the case?

Yes, I agree. Than the example code should probably just switch to `Counter`.

Regarding the polling approach, I was concerned that `getStats()` is called multiple times needlessly, but the overhead of this is probably negligible.

**Comment by ben-manes** [comment]
Oh yeah, `Counter` would have been more appropriate...
